resources/lang/zh/accounts.php

```diff
@@ -20,7 +20,7 @@
 
 return [
     'edit' => [
-        'title' => '账户设置',
+        'title' => '<strong>账户</strong>设置',
```

Comment:
is this actually valid in the chinese language? i know in japanese you would never use this style of bolding in headers (it would just look wrong)